dvc/dependency/__init__.py

```diff
@@ -19,6 +20,7 @@
 
 DEPS = [
     AzureDependency,
+    GDriveDependency,
```

Comment:
is it actually needed to support open?

Reply:
No - may have to revert `dependency` things for a different PR depending on @efiop feedback.

Reply:
@casperdcl `get_file_checksum` is not implemented for GdriveRemote, so this is not needed.